ign_rocker/templates/ignition_snippet.Dockerfile.em

```diff
@@ -0,0 +1,34 @@
+RUN apt-get -qq update && apt-get -y --no-install-recommends install \
+    cppcheck \
+    doxygen \
+    firefox \
+    gnupg \
+    g++-8 \
+    lsb-release \
+    wget \
+  && rm -rf /var/lib/apt/lists/*
+
+# set up repositories for Ignition and colcon
+# (adding nightlies just in case users want the newest Ignition version,
+# which is Edifice at the time of this writing)
+RUN sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list' \
+  && sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-nightly `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-nightly.list' \
+  && wget http://packages.osrfoundation.org/gazebo.key -O - | apt-key add - \
+  && sh -c 'echo "deb http://packages.ros.org/ros/ubuntu `lsb_release -cs` main" > /etc/apt/sources.list.d/ros-latest.list' \
+  && apt-key adv --keyserver 'hkp://keyserver.ubuntu.com:80' --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
+
+# install colcon (timezone must be enabled)
+RUN echo 'Etc/UTC' > /etc/timezone && ln -s /usr/share/zoneinfo/Etc/UTC /etc/localtime \
```

Comment:
Failed in local test due to the symlink already exist (did not investigate further):

```suggestion
RUN echo 'Etc/UTC' > /etc/timezone && ln -sf /usr/share/zoneinfo/Etc/UTC /etc/localtime \
```

Reply:
It's weird that this was an issue for you (I've never seen this happen), but I don't think there's anything wrong with adding the `-f` flag, so I added this in https://github.com/adlarkin/ign-rocker/pull/1/commits/136f9e901543419c5e3d597e43429d0afbeeb39f.